app/forms/concept_form.rb

```diff
@@ -0,0 +1,13 @@
+class ConceptForm
```

Comment:
@lsitu @hweng I'm fine with that.  I think that's a good design pattern that they've been using.